message/validation/validation.go

```diff
@@ -305,6 +273,24 @@ func (mv *messageValidator) validateP2PMessage(pMsg *pubsub.Message, receivedAt
 	defer mv.metrics.ActiveMsgValidationDone(topic)
 
 	messageData := pMsg.GetData()
+
+	var signatureVerifier func() error
+
+	currentEpoch := mv.netCfg.Beacon.EstimatedEpochAtSlot(mv.netCfg.Beacon.EstimatedSlotAtTime(receivedAt.Unix()))
+	if mv.netCfg.RSAMessageFork(currentEpoch) {
```

Comment:
Check my comment on `RSAMessageFork`, this can just be:
```
	if currentEpoch >= mv.netCfg.RSAForkEpoch {
```

Reply:
I agree no reason to add the func to networkconfig

Reply:
done